packages/server/lib/gui/auth.js

```diff
@@ -58,7 +69,7 @@ const _getOriginFromUrl = (originalUrl) => {
 /**
  * @returns a promise that is resolved with a user when auth is complete or rejected when it fails
  */
-const start = (onMessage) => {
+const start = (onMessage, utm_code) => {
```

Comment:
Keep it camelCase until it absolutely doesn't have to be:

```suggestion
const start = (onMessage, utmCode) => {
```